test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -219,7 +222,9 @@ private void testLocal_triggerDump() throws FileNotFoundException, IOException {
 	 *
 	 */	
 	@Test
-	private void testLocal_triggerDumpToFile() throws InvalidOptionException, FileNotFoundException, IOException {
+	private void testLocal_triggerDumpToFile() throws InvalidOptionException, FileNotFoundException, IOException, ClassNotFoundException, NoSuchMethodException,
+										IllegalAccessException, InvocationTargetException {
```

Comment:
The list of exceptions is rather long: is there value above just saying `throws Exception`?